src/main/resources/db/migration/V009__CREATE_USER.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE user
+(
+    id          BIGINT          NOT NULL,
+    username    VARCHAR(255)    NOT NULL    UNIQUE,
+    email       VARCHAR(255)    NOT NULL    UNIQUE,
+    password    VARCHAR(255)    NOT NULL,
+    active      BOOLEAN,
```

Comment:
What's the active column for?

Reply:
Its to check whether an account has been activated, like via an activation email

Reply:
Ah, I see